apps/system-e2e/src/support/commands.ts

```diff
@@ -75,7 +79,16 @@ Cypress.Commands.add(
 
 Cypress.Commands.add(
   'idsLogin',
-  ({ phoneNumber, authUrl, baseUrl, urlPath = '/' }) => {
+  ({ phoneNumber, authUrl, baseUrl, urlPath = '/', setBaseUrl = true }) => {
+    if (setBaseUrl) {
+      // FIXME: https://github.com/island-is/island.is/issues/8261
+      if (Cypress.env().testEnvironment == 'local') {
+        const newBaseUrl = getEnvironmentBaseUrl(baseUrl)
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        Cypress.config('baseUrl', newBaseUrl)
```

Comment:
are you sure you can change the baseUrl in a `beforeEach` call?
 
 I think this needs to happen in cypress config.

Reply:
Nope. This is leftovers from when I changed the port to 4242. The non-4200 port issue should be fixed seperately.

Reply:
Removed.